nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/processors/script/InvokeScriptedProcessor.java

```diff
@@ -92,11 +92,10 @@
                     logger.error(message, t);
                 }
             }
-        } else {
-            // Return defaults for now
-            relationships.add(REL_SUCCESS);
-            relationships.add(REL_FAILURE);
         }
+        // Add defaults
+        relationships.add(REL_SUCCESS);
+        relationships.add(REL_FAILURE);
```

Comment:
In ExecuteScript, the two relationships success and failure are defined.

In this case, the original code is adding, by default, the two relationships if and only if the processor didn't add its own relationships. With the change you propose, success and failure will be added in any case even if we don't want success/failure, but rather relationships "a", "b", "c". I think that the failing tests should be modified to add the relationships success/failure. At least if I am not missing something here...